run_lmod_cache.py

```diff
@@ -121,7 +122,18 @@ def main():
         'freshness-threshold': ('The interval in minutes for how long we consider the cache to be fresh',
                                 'int', 'store', 60*4),  # cron runs every 3 hours
         'module-basedir': ('Specify the base dir for the modules', 'str', 'store', MODULES_BASEDIR),
+        'check-cache-age': ('Show age in minutes of oldest cache and exit', None, 'store_true', False),
     }
+    opts = ExtendedSimpleOption(options, run_prologue=False)
```

Comment:
why are you adding this again?

Reply:
replaced with SimpleOption to avoid running this twice